homeassistant/components/advantage_air/switch.py

```diff
@@ -62,6 +65,32 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         await self.async_update_ac({"freshAirStatus": ADVANTAGE_AIR_STATE_OFF})
 
 
+class AdvantageAirMyFan(AdvantageAirAcEntity, SwitchEntity):
+    """Representation of Advantage Air MyFan control."""
+
+    _attr_icon = "mdi:fan-auto"
+    _attr_name = "MyFan"
+    _attr_device_class = SwitchDeviceClass.SWITCH
```

Comment:
Maybe set this entity to have entity category config?

https://developers.home-assistant.io/docs/core/entity#registry-properties